designs/senya/src/front.mjs

```diff
@@ -0,0 +1,159 @@
+import { base } from './base.mjs'
+import { bottomAndSide, sleevecap, shoulderAndNeck } from './shared.mjs'
+
+function senyaFront({
+  sa,
+  Point,
+  points,
+  Path,
+  paths,
+  Snippet,
+  snippets,
+  options,
+  measurements,
+  macro,
+  part,
+}) {
+  points.waistlineCp2 = points.waistline.shift(90, points.armhole.dy(points.waistline) / 3)
+
+  // Clone cb (center back) into cf (center front)
+  for (let key of ['Neck', 'Shoulder', 'Armhole', 'Waistline']) {
+    points[`cf${key}`] = points[`cb${key}`].clone()
+  }
+
+  // Neckline
+  points.cfNeck = new Point(0, options.necklineDepth * measurements.waistToUnderbust)
+  points.cfNeckCp1 = points.cfNeck.shift(0, points.neck.x * options.necklineBend * 2)
+  points.neck = points.hps.shiftFractionTowards(points.shoulder, options.necklineWidth)
+  points.neckCp2 = points.neck
+    .shiftTowards(points.shoulder, points.neck.dy(points.cfNeck) * (0.2 + options.necklineBend))
+    .rotate(-90, points.neck)
```

Comment:
There is something about the way `points.neckCp2` is calculated that causes the neckline curve seam allowance to be misshapen at times.
![Screenshot 2024-01-20 at 6 09 43 AM](https://github.com/freesewing/freesewing/assets/109869956/e64b7d90-f57b-4fde-9cf3-da40924ba9f5)
![Screenshot 2024-01-20 at 6 11 07 AM](https://github.com/freesewing/freesewing/assets/109869956/48c7ba5d-93b6-44f9-b86d-8730d4ce0f2f)